.ci/build-and-test.sh

```diff
@@ -42,3 +42,4 @@ set -euv
 ./b2 -j $JOBS $B2_OPTIONS toolset=$TOOLSET variant=$VARIANT cxxstd=$CXXSTD libs/gil/test/extension/numeric
 ./b2 -j $JOBS $B2_OPTIONS toolset=$TOOLSET variant=$VARIANT cxxstd=$CXXSTD libs/gil/test/extension/toolbox
 ./b2 -j $JOBS $B2_OPTIONS toolset=$TOOLSET variant=$VARIANT cxxstd=$CXXSTD libs/gil/test/extension/io//simple
+./b2 -j $JOBS $B2_OPTIONS toolset=$TOOLSET variant=$VARIANT cxxstd=$CXXSTD libs/gil/test/image_processing
```

Comment:
@simmplecoder It is yet to be argued between me and @stefanseefeld  about the `test/image_processing` location in near future. 

Meanwhile, I'd suggest you to do this:

1. Move `test/image_processing` into `test/core/image_processing`.
2. Remove this `b2` invocation from ` .ci/build-and-test.sh`
3. Enjoy the fact that your tests will be built automatically, *without* tweaking any CI configs.
4. Enjoy the fact your tests will be built *before* extensions, as they should be :)